middleware.js

```diff
@@ -186,7 +186,7 @@ module.exports = function(compiler, options) {
 					res.setHeader(name, options.headers[name]);
 				}
 			}
-			res.send(content);
+			res.end(content);
```

Comment:
What about a guard that defaults to the original implementation when Express isn't used (`karma-webpack` as an example) 
```
if (res.send) res.send(content);
else res.end(content);
```